acme_tiny.py

```diff
@@ -110,6 +109,8 @@ def _send_signed_request(url, payload):
         wellknown_path = os.path.join(acme_dir, token)
         with open(wellknown_path, "w") as wellknown_file:
             wellknown_file.write(keyauthorization)
+        if remote:
+            subprocess.Popen(["scp", wellknown_path, remote]).communicate()
```

Comment:
I'd like to get some comments on whether this works with password authentication and how error reporting works out. Key-based authentication works fine for me.